tests/unit/functions.tcl

```diff
@@ -288,6 +288,28 @@ start_server {tags {"scripting"}} {
         r function flush async
         assert_match {} [r function list]
 
+        # LAZYFREE_THRESHOLD is 64
```

Comment:
maybe we better extract all of this to a specific test for async flush? (not mixed with the normal flush test)

Reply:
done with [`8f5fe92` (#13383)](https://github.com/redis/redis/pull/13383/commits/8f5fe92b01b44534a7111c215799bc9b886ae320)